diff --git a/src/main.c b/src/main.c
index 5686f5b..98899fb 100644
--- a/src/main.c
+++ b/src/main.c
@@ -513,9 +513,6 @@ int main(int argc, char **argv)
 			      &ui.sensors_mutex,
 			      config_get_slog_interval());
 
-	ui_status_init(&ui);
-	ui_status_set_visible(1);
-
 	/* main window */
 	ui_window_create(&ui);
 
@@ -530,21 +527,9 @@ int main(int argc, char **argv)
 
 	g_timeout_add(1000 * ui.graph_update_interval, ui_refresh_thread, &ui);
 
-	ui_appindicator_init(&ui);
-	ui_unity_init();
-
 	gdk_notify_startup_complete();
 
-	/*
-	 * hack, did not find a cleaner solution.
-	 * wait 30s to ensure that the status icon is attempted to be
-	 * drawn before determining whether the main window must be
-	 * show.
-	 */
-	if  (ui.config->hide_on_startup)
-		g_timeout_add(30000, (GSourceFunc)initial_window_show, &ui);
-	else
-		initial_window_show(&ui);
+        initial_window_show(&ui);
 
 	/* main loop */
 	gtk_main();
