diff --git a/thumbs.c b/thumbs.c
index bfce2ff..1b19ef2 100644
--- a/thumbs.c
+++ b/thumbs.c
@@ -102,7 +102,7 @@ void tns_cache_write(Imlib_Image im, const char *filepath, bool force)
 					imlib_image_set_format("png");
 				} else {
 					imlib_image_set_format("jpg");
-					imlib_image_attach_data_value("quality", NULL, 90, NULL);
+					imlib_image_attach_data_value("quality", NULL, 99, NULL);
 				}
 				imlib_save_image_with_error_return(cfile, &err);
 			}
