diff --git a/data/ui/prefdlg.ui b/data/ui/prefdlg.ui
index 50ba3c6..3988d93 100644
--- a/data/ui/prefdlg.ui
+++ b/data/ui/prefdlg.ui
@@ -27,6 +27,9 @@
         <col id="0">192x192</col>
       </row>
       <row>
+        <col id="0">160x160</col>
+      </row>
+      <row>
         <col id="0">128x128</col>
       </row>
       <row>
@@ -79,6 +82,9 @@
         <col id="0">192x192</col>
       </row>
       <row>
+        <col id="0">160x160</col>
+      </row>
+      <row>
         <col id="0">128x128</col>
       </row>
       <row>
diff --git a/data/ui/prefdlg2.ui b/data/ui/prefdlg2.ui
index 4619fa5..65aa4ee 100644
--- a/data/ui/prefdlg2.ui
+++ b/data/ui/prefdlg2.ui
@@ -27,6 +27,9 @@
         <col id="0">192x192</col>
       </row>
       <row>
+        <col id="0">160x160</col>
+      </row>
+      <row>
         <col id="0">128x128</col>
       </row>
       <row>
@@ -79,6 +82,9 @@
         <col id="0">192x192</col>
       </row>
       <row>
+        <col id="0">160x160</col>
+      </row>
+      <row>
         <col id="0">128x128</col>
       </row>
       <row>
diff --git a/src/pref-dialog.c b/src/pref-dialog.c
index e1c2d37..f1dc07a 100755
--- a/src/pref-dialog.c
+++ b/src/pref-dialog.c
@@ -109,8 +109,8 @@ static const int tool_icon_sizes[] = {
     GTK_ICON_SIZE_DIALOG };
 // also change max_icon_size in settings.c & lists in prefdlg.ui prefdlg2.ui
 // see create_size in vfs-thumbnail-loader.c:_vfs_thumbnail_load()
-static const int big_icon_sizes[] = { 512, 384, 256, 192, 128, 96, 72, 64, 48, 36, 32, 24, 22 };
-static const int small_icon_sizes[] = { 512, 384, 256, 192, 128, 96, 72, 64, 48, 36, 32, 24, 22, 16, 12 };
+static const int big_icon_sizes[] = { 512, 384, 256, 192, 160, 128, 96, 72, 64, 48, 36, 32, 24, 22 };
+static const int small_icon_sizes[] = { 512, 384, 256, 192, 160, 128, 96, 72, 64, 48, 36, 32, 24, 22, 16, 12 };
 static const char* date_formats[] =
 {
     "%Y-%m-%d %H:%M",
