diff --git a/src/vfs/vfs-thumbnail-loader.c b/src/vfs/vfs-thumbnail-loader.c
index fb85499..62f0c98 100644
--- a/src/vfs/vfs-thumbnail-loader.c
+++ b/src/vfs/vfs-thumbnail-loader.c
@@ -477,7 +477,7 @@ static GdkPixbuf* _vfs_thumbnail_load( const char* file_path, const char* uri,
             if ( video_thumb )
             {
                 video_thumb->seek_percentage = 25;
-                video_thumb->overlay_film_strip = 1;
+                video_thumb->overlay_film_strip = 0;
                 video_thumb->thumbnail_size = create_size;
                 video_thumbnailer_generate_thumbnail_to_file( video_thumb,
                                                 file_path, thumbnail_file );
