diff --git a/src/vfs/vfs-thumbnail-loader.c b/src/vfs/vfs-thumbnail-loader.c
index fb85499..3b3f5b9 100644
--- a/src/vfs/vfs-thumbnail-loader.c
+++ b/src/vfs/vfs-thumbnail-loader.c
@@ -417,9 +417,24 @@ static GdkPixbuf* _vfs_thumbnail_load( const char* file_path, const char* uri,
 #endif
     strcpy( ( file_name + 32 ), ".png" );
 
-    thumbnail_file = g_build_filename( g_get_home_dir(),
-                                       ".thumbnails/normal",
-                                       file_name, NULL );
+    if ( create_size <= 128 )
+    {
+        thumbnail_file = g_build_filename( g_get_home_dir(),
+                                           ".thumbnails/normal",
+                                        file_name, NULL );
+    }
+    else if ( create_size <= 256 )
+    {
+        thumbnail_file = g_build_filename( g_get_home_dir(),
+                                           ".thumbnails/large",
+                                        file_name, NULL );
+    }
+    else
+    {
+        thumbnail_file = g_build_filename( g_get_home_dir(),
+                                           ".thumbnails/huge",
+                                        file_name, NULL );
+    }
 
     if( G_UNLIKELY( 0 == mtime ) )
     {
